### PR TITLE
fix: WEB-1805 Fixed JQL wrong filter pattern

### PR DESCRIPTION
**Pull request by BENBEN5959**
fix JQL Query

### CODE REVIEW

**Review by bonfim-sanofi — CHANGES_REQUESTED** [review]
Add tests for this bug.

File: src/proxy-page/steps/addJira.ts [hunk 1]
```diff
@@ -99,9 +99,12 @@ export default (config: ConfigService, jiraService: JiraService): Step => async
     const wikimarkup: string = elementJira.attribs['data-wikimarkup'];
     const xmlWikimarkup = cheerio.load(wikimarkup, { xmlMode: true });
     const server = xmlWikimarkup('ac\\:parameter[ac\\:name="server"]').text();
-    const filter = xmlWikimarkup(
+    let filter = xmlWikimarkup(
       'ac\\:parameter[ac\\:name="jqlQuery"]',
     ).text();
+    if (filter.includes('key')) {
+      filter = filter.replace('=', 'in');
```
Comment (krystiankedra) [inline src/proxy-page/steps/addJira.ts]: Change only filter value if starts from key =

File: .eslintrc.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ module.exports = {
         'no-underscore-dangle': 'off', // necessary for graphql convention
         'no-use-before-define': 'off', // idac habits
         'no-useless-constructor': 'off', // idac habits
+        'linebreak-style': 'off',
```
Comment (krystiankedra) [inline .eslintrc.js]: Please remove it.

File: tests/e2e/cypress/integration/jira.snapshot.spec.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/// <reference types="cypress" />
+
+context('User profile', () => {
+    const JIRA_URL = [
+      '/wiki/spaces/KONVIW/pages/64075268842/Jira+Snapshot+with+JQL+Query+key',
```
Comment (krystiankedra) [inline tests/e2e/cypress/integration/jira.snapshot.spec.ts]: In which tenant You created those pages ? Konviw or Sanofi ?

Comment (BENBEN5959) [reply tests/e2e/cypress/integration/jira.snapshot.spec.ts]: Sanofi confluence

**Review by krystiankedra — APPROVED** [review]

**Review by krystiankedra — APPROVED** [review]

**Pull request merged**